app/src/test/java/dev/hossain/weatheralert/data/WeatherRepositoryTest.kt

```diff
@@ -270,6 +270,31 @@ class WeatherRepositoryTest {
             assertThat(forecast.rain.nextDayRain).isEqualTo(5.9)
         }
 
+    @Test
+    fun `given weather response for oshawa - provides success response with parsed data`() =
+        runTest {
+            mockWebServer.enqueue(
+                MockResponse()
+                    .setResponseCode(200)
+                    .setBody(loadJsonFromResources("open-weather-oshawa-snow-fall.json")),
+            )
+
+            val result =
+                weatherRepository.getDailyForecast(
+                    cityId = 1,
+                    latitude = 0.0,
+                    longitude = -0.0,
+                )
+            assert(result is ApiResult.Success)
+            val forecast: ForecastData = (result as ApiResult.Success).value
+            assertThat(forecast.latitude).isEqualTo(43.9)
+            assertThat(forecast.longitude).isEqualTo(-78.85)
+            assertThat(forecast.snow.dailyCumulativeSnow).isEqualTo(5.89)
```

Comment:
Adding everything up for next 6 hours gives 5cm = 50mm .. and here it's just 5.89mm -- wtf? 
Did they get the units wrong?